src/libraries/System.Runtime.Numerics/src/System/Numerics/BigInteger.cs

```diff
@@ -3481,42 +3481,20 @@ public static BigInteger TrailingZeroCount(BigInteger value)
 
             ulong result = 0;
 
-            if (value._sign >= 0)
-            {
-                // When the value is positive, we simply need to do a tzcnt for all bits until we find one set
-
-                uint part = value._bits[0];
+            // As explained in https://github.com/dotnet/runtime/issues/77720, both positive values and
```

Comment:
Nit: I don't think we need the "As explained in https://github.com/dotnet/runtime/issues/77720," part